### ISSUE TITLE
Error: The argument type 'bool?' can't be assigned to the parameter type 'bool' because 'bool?' is nullable and 'bool' isn't.

### ISSUE THREAD

**Issue by Vito0912** [stryder-dev/flutter_platform_widgets#384]
I created a new Project.

Currently only using following dependecies:
  flutter_adaptive_ui: ^0.8.0+1
  flutter_platform_widgets: ^3.0.0
  flutter_bloc: ^8.1.1
  
Now I get the error:
 `platform.dart:297:29: Error: The argument type 'bool?' can't be assigned to the parameter type 'bool' because 'bool?' is nullable and 'bool' isn't.`
 
 Is there a fix or do I forgot something?

**Comment by Aqua-Ye** [comment]
I have the same issue with dart 3. 

```
../../.pub-cache/hosted/pub.dev/flutter_platform_widgets-3.0.0/lib/src/platform.dart:297:29: Error: The argument type 'bool?' can't be assigned to the parameter type 'bool' because 'bool?' is nullable and 'bool' isn't.
      semanticsDismissible: cupertino?.semanticsDismissible,
```

```
[!] Flutter (Channel master, 3.8.0-5.0.pre.1, on macOS 13.1 22C65 darwin-arm64, locale en-FR)
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision bbca7ff693 (5 hours ago), 2023-02-06 11:02:37 +0200
    • Engine revision 2a104cdfcd
    • Dart version 3.0.0 (build 3.0.0-204.0.dev)
    • DevTools version 2.21.1
```

**Comment by Vito0912** [comment]
I have found a work-around. 
Just switch the channel to "stable" and try again. (downgrade to Flutter 3.7.x)
As the stated in there version history of "flutter_platform_widgets", they upgraded only to Flutter 3.7 - [Version](https://github.com/stryder-dev/flutter_platform_widgets/commit/d1bcbb4903eff3790673748689f724eb7d22a477)
So there will probably be some "fixed" version for Flutter 3.8.x

Hope I could help
